File: src/main/webapp/app/Konsulent.js
function Konsulent(data) {
    "use strict";
    var self = this;
    self.id = ko.observable();
    self.navn = ko.observable();
    self.epost = ko.observable();
    self.image_url = ko.observable();
    self.pris = ko.observable();
    self.stilling = ko.observable();

    if(data != undefined && data != null){
        self.id(data.id);
        self.navn(data.navn);
        self.epost(data.epost);
        self.image_url(data.image_url);
        self.pris(data.pris);
        self.stilling(data.stilling);
    }


}

File: src/main/java/com/soprasteria/boot/domain/Konsulent.java
package com.soprasteria.boot.domain;

import org.springframework.context.annotation.Bean;
import org.springframework.stereotype.Component;

@Component
public class Konsulent {
	private String navn; 
	private int id; 
	private String epost; 
	private String image_url;
	private int pris; 
	private Stilling stilling;

	public Konsulent() {}

	public Konsulent(int id) {
		this.id = id;
	}
	
	
	
	public String getNavn() {
		return navn;
	}
	public void setNavn(String navn) {
		this.navn = navn;
	}
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getEpost() {
		return epost;
	}
	public void setEpost(String epost) {
		this.epost = epost;
	}
	public String getImage_url() {
		return image_url;
	}
	public void setImage_url(String image_url) {
		this.image_url = image_url;
	}
	public int getPris() {
		return pris;
	}
	public void setPris(int pris) {
		this.pris = pris;
	}
	public Stilling getStilling() {
		return stilling;
	}
	public void setStilling(Stilling stilling) {
		this.stilling = stilling;
	} 
	
	
	
	
	

}
